### ISSUE TITLE
Dev (master) tagged as v1.10.5+23-g6618348e2 when compiling

### ISSUE THREAD

**Issue by iamdoubz** [go-gitea/gitea#10707]
- Gitea version (or commit ref): 1.12.0-dev+4something (but displays as 1.10.5+23-g6618348e2)
- Git version: 2.25.1
- Operating system: Windows, but all of them
- Database (use `[x]`):
  - [X] PostgreSQL
  - [X] MySQL
  - [X] MSSQL
  - [X] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [X] Not relevant
- Log gist:

```
git checkout master
git pull
iamdoubz@comp MINGW64 ~/Gits/gitea (master)
$ TAGS="bindata sqlite sqlite_unlock_notify" make build
...
writing bindata.go
GO111MODULE=on go build -mod=vendor -v  -tags 'bindata sqlite sqlite_unlock_notify' -ldflags '-s -w  -X "main.MakeVersion=GNU Make 4.3" -X "main.Version=1.10.5+23-g6618348e2" -X "main.Tags=bindata sqlite sqlite_unlock_notify"' -o gitea.exe
github.com/src-d/enry/v2/data/rule
github.com/toqueteos/trie
golang.org/x/crypto/internal/subtle
...
```
## Description

Opposite of re-release of 1.10.5 -> 1.10.6. Easy fix and I'm sure you would have seen it tomorrow.

## Screenshots

N/A

**Comment by lunny** [comment]
This will be resolved after we delete the release v1.10.5

**Comment by lunny** [comment]
I think this has been resolved since v1.10.5 has been replaced v1.10.6
